setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="filebrowser_safe",
-    version="0.5.0bevy2",
+    version="0.5.0.post3",
```

Comment:
Curious: what is this version convention based on?

Reply:
I learned, in trying to make the warnings about wrong dependency versions for Mezzanine go away, that Python packages have their own versioning scheme, specified in [PEP 440](https://www.python.org/dev/peps/pep-0440/#post-releases).

Based on experimentation, a post release version was the only way to increment the version, but also satisfy the dependency on `>=0.5.0` without a likely collision with a real-live release version. I wanted to get `bevy` in the version string but it gave warnings every time. Perhaps I didn't read the PEP carefully enough...